### PR TITLE
Dev jg

### PR DESCRIPTION
**Pull request by jorgitogb**

### CODE REVIEW

File: config.yml
Comment (Zalfsten) [inline config.yml:1]: Please remove the comments for bonares and edal

File: scrap_repo_test.ipynb
Comment (Zalfsten) [inline scrap_repo_test.ipynb:1]: please not not commit this file

File: src/sitemap_parser_openagrar.py
Comment (Zalfsten) [inline src/sitemap_parser_openagrar.py:1]: please remove unneeded imports.

**Review by Zalfsten — CHANGES_REQUESTED** [review]

File: config.yml
Comment (jorgitogb) [reply config.yml:1]: done

File: scrap_repo_test.ipynb
Comment (jorgitogb) [reply scrap_repo_test.ipynb:1]: done

**Pull request merged**